### ISSUE TITLE
[BUG?] Message vanishs if composed throughout a message search

### ISSUE THREAD

**Issue by therob84** [qklabs/qksms#398]
Today I used the search function to find a message with a contact.
(basically I looked only for the contact-history of this contact, but the search for contacts is still missing in qksms...),
Anyway: I selected one message of the search results of the desired contacts and could compose a new message to this contact. In this normal compose window only the last message was displayed, but the contacts name was in the title - so everything seemed OK.

I hit send and the message seemed to be sent. 
**But:** I can't find this message anymore, neither in the conctac's conversation, nor in the message-overview, nor in the standard Android SMS-App, nor with the search function.

So, where is it? Was is sent? Why I cant find it anymore? I did not get any error message....
What's happening here?

Can anybody reproduce this?

**Comment by moezbhatti** [comment]
If you try it again, does the message still disappear?

**Comment by therob84** [comment]
Ok just tried again, and it happened something slightly different:

**Desription**:
- I have a dual SIM, so e sent from SIM1 to SIM2 an SMS with "TEST123"
- I searched for "TEST123" (as I cant search for contacts, #396) and found two messages (the sent and the received one)
- I clicked the received-one --> the compose window appears --> BUT: the conversation belonging to this number (SIM2) is not displayed - I see an empty history (see #397). In the "start screen" the history is there!
- I compose a new message and sent again from SIM1 to SIM2 with "Test987"
- If I click send the **error message** appears "**Chose to which contacts the message should be send**" (translated from german) although the contacts name (my SIM2) stands in the headline of the (empty) conversation, but it is not sent! There is also no field to add contacts!
- If I go "back", the message is **NOT saved** as draft (see here #314)

**Remarks**:
- I tried the same with a different number of another contact (to exclude the problem connected with sending only to my own SIM2) --> and the same happened!
- I guess at the first attempted in the initial post I just did not noticed the error message, and switched back to the normal view. So it was not sent, I did not noticed the error message, and by going back it is not saved as draft --> this would explain all.

**So with this I really ask you to solve these problems and all ticket#s I linked here!**
Before unfortunately I can't recommend qksms to my friends for using it.....and myself are not very comfortable with using it!

Best, Robert
